rfc/multiple-go-modules.md

```diff
@@ -0,0 +1,187 @@
+| cli              | pkg/shared/job_manager        | cli/pkg/jobmanager               | cli/v0.7.0                              |
+| cli              | pkg/shared/scanner            | cli/pkg/scanner                  | cli/v0.7.0                              |
+| cli              | pkg/shared/utils              | cli/pkg/utils                    | cli/v0.7.0                              |
+| orchestrator     | pkg/orchestrator              | orchestrator                     | orchestrator/v0.7.0                     |
+| uibackend        | pkg/uibackend                 | uibackend                        | uibackend/v0.7.0                        |
+| uibackend/server | pkg/uibackend/rest            | uibackend/server                 | uibackend/server/v0.7.0                 |
+| uibackend/client | pkg/shared/uibackendclient    | uibackend/client                 | uibackend/client/v0.7.0                 |
+| utils            | pkg/version                   | utils/version                    | utils/v0.7.0                            |
+| utils            | pkg/shared/command            | utils/command                    | utils/v0.7.0                            |
+| utils            | pkg/shared/fsutils            | utils/fsutils                    | utils/v0.7.0                            |
+| utils            | pkg/shared/log                | utils/log                        | utils/v0.7.0                            |
+| utils            | pkg/shared/manifest           | utils/manifest                   | utils/v0.7.0                            |
+| provider         | pkg/orchestrator/provider     | provider                         | provider/v0.7.0                         |
+| provider         | pkg/containerruntimediscovery | provider/common/runtimediscovery | provider/common/runtimediscovery/v0.7.0 |
+
+To improve compliance with https://github.com/golang-standards/project-layout, the changes below are also proposed.
+
+| Module       | Current path                     | New path                      |
+| ------------ | -------------------------------- | ----------------------------- |
+| provider     | example_external_provider_plugin | provider/examples/external    |
+| cli          | scanner_boot_test                | cli/test/boot                 |
+
+Makefile, GitHub workflows and other files will need to be updated to point to the new paths.
+
+The VMClarity directory will have the following structure.
+
+```sh
+.
+├── Makefile
+├── api
+│   ├── client
+│   │   ├── client.cfg.yaml
+│   │   └── go.mod
+│   ├── go.mod
+│   ├── models
+│   │   └── models.cfg.yaml
+│   ├── openapi.yaml
+│   └── server
+│       ├── cmd
+│       ├── go.mod
+│       ├── pkg
+│       │   ├── common
+│       │   ├── database
+│       │   └── rest
+│       └── server.cfg.yaml
+├── cli
+│   ├── cmd
+│   ├── go.mod
+│   ├── pkg
+│   │   ├── analyzer
+│   │   ├── config
+│   │   ├── converter
+│   │   ├── families
+│   │   ├── findingkey
+│   │   ├── job_manager
```

Comment:
should be `jobmanager` since the table above lists it properly

Reply:
thanks for noticing this!